easybuild/easyblocks/g/gcc.py

```diff
@@ -203,6 +203,13 @@ def __init__(self, *args, **kwargs):
             self.log.warning('Setting withnvptx to False, since we are building on a RISC-V system')
             self.cfg['withnvptx'] = False
 
+        # define list of subdirectories in search paths of module load environment
+        self.module_load_environment.PATH = ['bin']
+        self.module_load_environment.LD_LIBRARY_PATH = ['lib', 'lib64']
```

Comment:
These two lines don't need to be here since framework takes care of this exactly like this

Reply:
fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/3584/files/04dc2b383f7116d218e7fe984ae47cd7ab063eed..9926576e2256d421d0a0cc768c2d6ca85d6b7119